### ISSUE TITLE
Sanctum issue ?

### ISSUE THREAD

**Issue by zelionix** [maxhoesel-ansible/ansible-collection-pterodactyl#223]
I succeed install on my debian 12 virtual machine.

`pterodactyl_panel_domain: "pterodactyl.local"`

I get the login page... Nice !
However when trying to login : 

`10.0.2.2 - - [22/Dec/2024:01:53:57 +0100] "GET /sanctum/csrf-cookie HTTP/1.1" 404 187 "http://pterodactyl.local:8001/auth/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"`

NB : I have a port forward  local port 8001 --> VM port 80

**Comment by zelionix** [comment]
Simple possible conflict with my vm port 
vendor/laravel/sanctum/config/sanctum.php:    'stateful' => explode(',', env('SANCTUM_STATEFUL_DOMAINS', sprintf(
vendor/laravel/sanctum/config/sanctum.php-        '%s%s',
vendor/laravel/sanctum/config/sanctum.php-        'localhost,localhost:3000,127.0.0.1,127.0.0.1:8000,::1',

**Comment by zelionix** [comment]
OK found the issue ! The stack is not working using **http only** protocol.
The application configuration in **/var/www/pterodactyl/.env** is set with 
`APP_URL={{ pterodactyl_panel_url | quote }}`
but the ansible defaults set : 
`pterodactyl_panel_url: "https://{{ ansible_fqdn }}"`

**Comment by maxhoesel** [comment]
Hi! This is intentional - the panel role is designed to only support access via HTTPS, not plain HTTP. See also the apache template: https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl/blob/main/roles/pterodactyl_panel/templates/pterodactyl.conf.j2

This information should probably be added to the README. I'm open to reworking the webserver integration in the future to be more generic and to support HTTP (also related to #220), but I don't actively use this collection myself anymore, so this is low-priority for me.

**Comment by zelionix** [comment]
Agreed this should be added in the Readme as this is "security by design".
Not sure a good idea to introduce the http support, as this not designed for... ^_ ^
Will add it in Readme in the PR concerning nginx.
